doc/source/main-documentation/qibolab.rst

```diff
@@ -168,22 +190,22 @@ The controller's driver ensures the correct pulse frequency is set based on the
 Let's explore an example using an RFSoC controller.
 Note that while channels are defined in a device-independent manner, the port parameter varies based on the specific instrument.
 
-.. code-block:: python
+.. testcode:: python
 
     from qibolab.channels import Channel, ChannelMap
     from qibolab.instruments.rfsoc import RFSoC
 
-    controller = RFSoC()
+    controller = RFSoC(name="dummy", address="192.168.0.10", port="6000")
     channel1 = Channel("my_channel_name_1", port=controller[1])
     channel2 = Channel("my_channel_name_2", port=controller[2])
     channel3 = Channel("my_channel_name_3", port=controller[3])
 
 Channels are then organized in :class:`qibolab.channels.ChannelMap` to be passed as a single argument to the platform.
 Following the tutorial in :doc:`/tutorials/lab`, we can continue the initialization:
 
-.. code-block:: python
+.. testcode:: python
 
-    from qibolab.serialize import load_qubits
+    from qibolab.serialize import load_qubits, load_runcard
 
     ch_map = ChannelMap()
     ch_map |= channel1
```

Comment:
This one fails because it tries to load a non-existing runcard. Should we leave it as `code-block` and just skip it?